src/monitor-pipeline-group/azext_monitor_pipeline_group/tests/latest/data_files/exporters.json

```diff
@@ -0,0 +1,29 @@
+[
+    {
+        "name": "gigla1",
+        "type": "AzureMonitorWorkspaceLogs",
+        "azureMonitorWorkspaceLogs": {
+            "api": {
+                "dataCollectionEndpointUrl": "https://logs-ingestion-eb0s.eastus-1.ingest.monitor.azure.com",
```

Comment:
temporary resource not recommend added in test data sets cause it may break live tests running.

Reply:
Replaced with mock endpoint and re-recorded the live test. This field doesn't need to be valid for resource creation to succeed.